tests/functional/search/test_index.py

```diff
@@ -40,14 +42,15 @@ def test_index_create(run_line):
             "example_cookery",
             "Example index of Cookery",
         ],
-        matcher=True,
+        match_out=("Index ID", index_id),
     )
-    matcher.check(r"^Index ID:\s+([\w-]+)$", groups=[index_id])
 
 
 def test_index_delete(run_line):
     meta = load_response_set("cli.search").metadata
     index_id = meta["index_id"]
 
-    result = run_line(f"globus search index delete {index_id}")
-    assert f"Index {index_id} is now marked for deletion." in result.output
+    run_line(
+        f"globus search index delete {index_id}",
+        match_out=f"Index {re.escape(index_id)} is now marked for deletion.",
```

Comment:
I think the `re.escape()` call is harmless, but I don't think it's needed -- UUIDs don't trigger special regex behavior.

Reply:
Yes, it is harmless... I waffled about whether or not to include it, since it's a no-op.
Why don't I remove it, now that it's come up?

I like `re.escape`-ing anything which looks like "input", but have no particular attachment to doing it in this case.